gremlin-console/src/main/groovy/org/apache/tinkerpop/gremlin/console/Console.groovy

```diff
@@ -191,7 +192,18 @@ class Console {
             groovy.setResultHook(handleResultShowNothing)
     }
 
-    private def handlePrompt = { interactive ? STANDARD_INPUT_PROMPT : "" }
+    private def handlePrompt = { 
+        if (interactive) {
+            int lineNo = groovy.buffers.current().size()
+            if (lineNo > 0 ) {
+                return lineNo.toString().padLeft(STANDARD_INPUT_PROMPT_PAD, '0').padRight(STANDARD_INPUT_PROMPT.length()-2, ' ') + "> "
```

Comment:
Yeah, it might be better to merge the 2 PRs together. `STANDARD_INPUT_PROMPT.length()` is replaced by the `input.prompt` preference from the colorization PR.